src/tools/doall/src/DOALL_chunking.cpp

```diff
@@ -118,6 +118,103 @@ void DOALL::rewireLoopToIterateChunks(LoopDependenceInfo *LDI) {
     this->IVValueJustBeforeEnteringBody[ivPHI] = chunkedIVValues;
   }
 
+  auto sccManager = LDI->getSCCManager();
+  auto sccdag = sccManager->getSCCDAG();
+
+  /*
+   * Generates code for periodic variable SCCs to match the DOALL chunking
+   * strategy.
+   */
+  for (auto scc : sccdag->getSCCs()) {
+    auto sccInfo = sccManager->getSCCAttrs(scc);
+    auto periodicVariableSCC = dyn_cast<PeriodicVariableSCC>(sccInfo);
+    if (periodicVariableSCC == nullptr)
+      continue;
+    // errs() << "DOALL: periodic variable with initial value " <<
```

Comment:
Please delete commented code (or enable it perhaps when the verbosity level is higher than 2)

Reply:
changed to match verbosity level in [557b357](https://github.com/arcana-lab/noelle/pull/74/commits/577b3570e8fec4c796a6d550d249cb6fcd689d34)